courts_db/data/courts.json

```diff
@@ -63252,6 +63252,26 @@
         "system": "state",
         "type": "appellate"
     },
+    {
+        "citation_string": "Tex. Bus.",
+        "dates": [
+            {
+                "end": null,
+                "start": "2024-10-30"
```

Comment:
Here I would say the court starts not at first publication but when it was established by Texas. 

`On June 9, 2023,` according to wikipedia